ci/environment.yml

```
@@ -41,4 +41,4 @@ dependencies:
   - pyspark ==3.4.1
   - openjdk ==20.0.2
   - python-duckdb ==0.9.1
-  - polars >=0.19.3,<=0.19.8  # 0.19.3 on Windows, 0.19.8 on Linux and MacOS
+  - polars >=0.19.3,<=0.19.9  # 0.19.3 on Windows, 0.19.8 on Linux and MacOS
```

Should these be kept in sync?
```suggestion
  - polars >=0.19.3,<=0.19.9  # 0.19.3 on Windows, 0.19.9 on Linux and MacOS
```